pkg/crd/testdata/cronjob_types.go

```diff
@@ -119,7 +119,7 @@ type CronJobSpec struct {
 
 	// +kubebuilder:validation:EmbeddedResource
 	// +kubebuilder:validation:nullable
-	EmbeddedResource runtime.RawExtension `json:"embeddedResource"`
```

Comment:
I thought this was testing an arbitrary embedded object, which would mean we should set `x-kubernetes-preserve-unknown-fields: true`, right?